pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2024,6 +2024,38 @@ var _ = Describe("Converter", func() {
 			Entry("use virtio transitional", true),
 			Entry("use virtio non-transitional", false),
 		)
+		DescribeTable("Should set the error policy", func(epolicy v1.ErrorPolicy, expected string, specified bool) {
+			v1Disk := &v1.Disk{
```

Comment:
nit - I'd just do this inline below and then modify the `ErrorPolicy` directly to save declaring `v1Disk`.